subworkflows/nf-core/fastq_align_mapad/main.nf

```diff
@@ -0,0 +1,81 @@
+//
+// Alignment with mapAD and sort
+//
+
+include { MAPAD_MAP               } from '../../../modules/nf-core/mapad/map/main'
+include { BAM_SORT_STATS_SAMTOOLS } from '../bam_sort_stats_samtools/main'
+
+workflow FASTQ_ALIGN_MAPAD {
+
+    take:
+    ch_reads // channel (mandatory): [ val(meta), path(reads) ]. Important: meta REQUIRES single_end` entry!
+    ch_index // channel (mandatory): [ val(meta), path(index) ]
+    ch_fasta // channel (optional) : [ val(meta3), path(fasta) ]
+    val_mismatch_parameter
+    val_double_stranded_library
+    val_five_prime_overhang
+    val_three_prime_overhang
+    val_deam_rate_double_stranded
+    val_deam_rate_single_stranded
+    val_indel_rate
```

Comment:
Only file channels should be taken in. Values can be supplied to modules using `ext.args` instead.

Reply:
apparently newer module guidelines allow values for required options, so nevermind that!